src/components/mortgage-investment-compare/input-card.js

```diff
@@ -197,13 +198,24 @@ const InputCard = props => {
         </FormGroup>
         <hr />
         <FormGroup className='mb-3'>
-          <Tooltip
-            icon={faQuestionCircle}
-            content='How well you expect your investment to perform each year'
-            id='investmentRate'
-          >
-            <label>Return on Investment (ROI)</label>
-          </Tooltip>
+          <div className='d-flex flex-row justify-content-between'>
+            <Tooltip
+              icon={faQuestionCircle}
+              content='How well you expect your investment to perform each year'
+              id='investmentRate'
+            >
+              <label>Return on Investment (ROI)</label>
+            </Tooltip>
+            <CustomInput
+              type='checkbox'
+              className='custom-switch'
+              id='roi-switch'
+              name='roi-switch'
+              label={<small>Include?</small>}
```

Comment:
I used `<small>` but still not 100% convinced with the font styling. If you can make it better please be my guest. If you like it like this then nbd.